include/core/utilities.h

```diff
@@ -615,4 +617,24 @@ compute_cell_diameter(const double cell_measure, const unsigned int fe_degree)
   return h;
 }
 
+/**
+ * @brief Computes the area (2D) or volume (3D) of the cell by integrating 1
+ * over the cell, by summing JxW values (quadrature weights) returned by the
+ * FEValues object.
+ *
+ * @tparam dim Number of spatial dimensions (2D or 3D).
+ *
+ * @param[in] JxW_values Vector of mapped quadrature weights.
+ *
+ * @return Area (2D) volume (3D) of the cell.
+ */
+inline double
+compute_volume_with_JxW(const std::vector<double> &JxW_values)
```

Comment:
```suggestion
compute_cell_measure_with_JxW(const std::vector<double> &JxW_values)
```

would be a better name IMO

Reply:
You're right, I missed it